Removing docker jmx FAQ. (#4641)
DataDog/documentation: content/en/integrations/faq/troubleshooting-jmx-integrations.md

| `sudo /etc/init.d/datadog-agent jmx list_everything`              | List every attributes available that has a type supported by JMXFetch.                                                                                                  |
| `sudo /etc/init.d/datadog-agent jmx collect`                      | Start the collection of metrics based on your current configuration and display them in the console.                                                                    |

{{% /tab %}}
{{% tab "Docker Agent" %}}

To check whether Autodiscovery is loading JMX-based checks:

```
docker exec -it <AGENT_CONTAINER_NAME> datadog-agent configcheck
```

To see JMX-based checks status from the Agent:

```
$ docker exec -it <AGENT_CONTAINER_NAME> datadog-agent status
```

{{% /tab %}}
{{< /tabs >}}

## FAQs

### The 350 metric limit

> Due to the nature of these integrations, it is possible to submit an extremely high number of metrics directly to Datadog, note that Datadog sets the limit at maximum 350 metrics.

Reviewer: ```suggestion
Datadog accepts a maximum of 350 metrics.
```

> It is recommended to create filters to refine what metrics are collected but if you believe you need more than 350 metrics, contact [Datadog support][3].

Reviewer: ```suggestion
A best practice is to limit your metrics to less than 350 by creating filters to refine those metrics collected, but if you need more than 350 metrics, contact [Datadog support][3].
```

* If JMXFetch is running on a different host than the JBoss/WildFly application, copy `jboss-cli-client.jar` to a location on the host JMXFetch is running on.
* Add the path of the jar to the `init_config` section of your configuration:

```yaml
  # Datadog Agent >= 5.6.0

  init_config:
    custom_jar_paths:
      - /path/to/jboss-cli-client.jar
```

* Specify a custom URL that JMXFetch connects to, in the `instances` section of your configuration:

  ```yaml
  # Datadog Agent >= 5.6.0

  # The jmx_url may be different depending on the version of JBoss/WildFly you're using
  # and the way you've set up JMX on your server
  # Refer to the relevant documentation of JBoss/WildFly for more information
  instances:
    - jmx_url: "service:jmx:remote://localhost:4447"
      name: jboss-application  # Mandatory, but can be set to any value,
                               # is used to tag the metrics pulled from that instance
  ```

* [Restart the Agent][4].

### Monitoring Tomcat with JMX Remote Lifecycle Listener enabled

The following instructions work on Agent v5.6.0+.

> If you're using Tomcat with JMX Remote Lifecycle Listener enabled (see the [Tomcat documentation][5] for more information), JMXFetch needs additional setup to be able to connect to your Tomcat application.

Reviewer: ```suggestion
If you're using Tomcat with JMX Remote Lifecycle Listener enabled (see the [Tomcat documentation][5] for more information), JMXFetch needs an additional setup to be able to connect to your Tomcat application.
```
